### PR TITLE
[XMToolbar] Brand status steady icon.

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/StockIcons.cs [hunk 1]
```diff
@@ -141,7 +141,6 @@ public class Stock
 		public static readonly IconId StatusUpload = "md-status-upload";
 		public static readonly IconId StatusSearch = "md-status-search";
 		public static readonly IconId StatusBuild = "md-status-build";
-		public static readonly IconId StatusSteady = "md-status-steady";
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/StockIcons.cs]: I could make this be BrandingService.StatusSteadyStockIconId and not break API and stuff.

Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/StockIcons.cs]: Fixed.

**Comment by slluis** [comment]
Looks good.

**Pull request merged**